pkgs/development/python-modules/packageurl-python/default.nix

```
@@ -0,0 +1,18 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  pname = "packageurl-python";
+  version = "0.9.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0mpvj8imsaqhrgfq1cxx16flc5201y78kqa7bh2i5zxsc29843mx";
+  };
+
+  meta = with lib; {
+    description = "A parser and builder for purl aka 'Package URLs' for Python 2 and 3.";
```

```suggestion
    description = "A parser and builder for purl aka 'Package URLs' for Python 2 and 3";
```

see https://nixos.org/manual/nixpkgs/unstable/#sec-standard-meta-attributes

Personally I would even go a step further and get rid of "purl" and the Python releases: 

```suggestion
    description = "Python parser and builder for package URLs";
```

---

This makes sense. Will change.